doc/source/releasenotes/1.14.0.rst

```diff
@@ -0,0 +1,81 @@
+====================
+1.14.0 "Professor D"
+====================
+
+-----
+Rucio
```

Comment:
Rucio Server


?
(as below we have clients)

Reply:
This is how we do it since a long time. These are the issues from the main package (1.15.0 instead of 1.15.0-clients or 1.15.0-webui) - The thing is server is not really correct, as it is also daemons and very often basically affects everything, also clients, thus calling it Server/Daemons is also not really correct. In the beginning I was thinking considering calling it core, but this is confusing so I just called it purely Rucio.

Reply:
"Main changes" ?